README.md

```diff
@@ -1 +1,130 @@
+
+This pipeline provides a comprehensive workflow for assembling long reads, supporting input files in FASTQ, BAM, or POD5 formats. The focus is on bacterial genomes but the pipeline can be also used for other species (untested). 
+
+⚠ Note: The pipeline processes only one input type at a time.
+
+For POD5 files, it uses [**Dorado**](https://github.com/nanoporetech/dorado) to basecall.
+
+![Alt text](images/LARRI_workflow.png)
+
+In the subway-style diagram above, the green track represents the default workflow. The pipeline is modular, so alternative tracks can be followed by enabling or disabling steps through parameters (see `--help`). It is also possible to run the pipeline just for basecalling, without performing the assembly steps.
+
+## How to Run
+
+To run **LARRI**, you must specify **exactly one input type**:  
+- a **BAM** file using the `--bam` option  
+- a **FASTQ** file using the `--fastq` option  
+- a **pod5** file or folder using the `--pod5` option  
+
+### Usage Examples
+
+**Attention:** We use release version 0.0.1 in these example commands but always check for the latest or matching release you want to run. You can use `nextflow info rki-mf1/LARRI` to screen available release versions.
+
+Run with a single BAM file:
+
+```bash
+# check for the release you want to use! 
+nextflow run rki-mf1/LARRI -r 0.0.1 --bam 'sample.bam'
+```
+
+Run with a single FASTQ file:
+
+```bash
+nextflow run rki-mf1/LARRI -r 0.0.1 --fastq 'sample.fastq.gz'
+```
+
+Run with multiple BAM or FASTQ files (using wildcard):
+
+```bash
+nextflow run rki-mf1/LARRI -r 0.0.1 --bam '*.bam'
+nextflow run rki-mf1/LARRI -r 0.0.1 --fastq '*.fastq.gz'
+```
+
+Run with a single pod5 file or a folder:
+
+```bash
+nextflow run rki-mf1/LARRI -r 0.0.1 --pod5 'file.pod5'
+nextflow run rki-mf1/LARRI -r 0.0.1 --pod5 '/path/to/folder/'
+```
+
+> **Important**:
+> - Only one of the options `--bam`, `--fastq`, or `--pod5` can be provided at a time. 
+> - Multiple input files can only be used with `--bam` and `--fastq` (via wildcards). Use single ticks! 
+> - The `--pod5` option only supports a single file or a folder of files.  
+
+
+## Running with Containers
+
+Run using **Docker**:
+
+```bash
+nextflow run rki-mf1/LARRI -r 0.0.1 --bam 'sample.bam' -profile docker
+```
+
+Run with **SLURM** and **conda** (currently not supported, but coming soon):
+```bash
+nextflow run rki-mf1/LARRI -r 0.0.1 --bam 'sample.bam' -profile slurm,conda
+```
+
+Run with **SLURM** and **Singularity**:
+
+```bash
+nextflow run rki-mf1/LARRI -r 0.0.1 --bam 'sample.bam' -profile slurm,singularity
+```
+
+## Dorado Basecalling
+
+If pod5 files are provided as input, the pipeline will use **Dorado** to basecall the data.  
+
+To also run [Dorado demultiplexing](https://github.com/nanoporetech/dorado?tab=readme-ov-file#barcode-classification), pass the `--demux` parameter. In this case:  
+- The Dorado basecaller will **not trim the barcodes**.  
```

Comment:
why not? Should we not activate trimming?

Reply:
Ahhh true! I somehow had that back in my mind but thx for the reminder. That's really an important thing and actualyl smt we should benchmark at some point... to see how large the effect is. But then it's good to have it in the pipeline like that.